torch/_inductor/scheduler.py

```diff
@@ -77,6 +78,9 @@ class SchedulerBuffer:
     node: ir.Buffer
     defining_op: BaseSchedulerNode
     users: List[NodeUser] = dataclasses.field(default_factory=list)
+    mpi: MemoryPlanningInfoForBuffer = dataclasses.field(
```

Comment:
I believe `.mpi` can mean two things in this PR: `MemoryPlanningInfoForBuffer` or `MemoryPlanningInfoForNode`. I wonder is there a way to easily differentiate the two? (maybe we can call them `mpi_buffer` and `mpi_node`?)

Reply:
changed to mpi_buffer and mpi_node depending on the type.